src/glyph/parse.rs

```diff
@@ -156,6 +204,50 @@ impl<'names> GlifParser<'names> {
         Ok(())
     }
 
+    fn parse_point<'a>(
```

Comment:
was this just moved, or was the code changed?

As patches get bigger it's nice sometimes to avoid making non-functional changes (like moving code) since it adds to the diff and can make the patch harder to reason about; just something I try to keep in mind.

Reply:
I just moved it! Ok, I can do a clean-up run after I fleshed this out. See it as a draft.